<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1363</width>
    <height>792</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1363</width>
    <height>792</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1363</width>
    <height>792</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>mini-PS</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/main.png</normaloff>icons/main.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
color: rgb(0, 0, 0);

	background-color:#191934;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>1221</width>
      <height>841</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(24,24,36);
border-color: rgb(16, 20, 20);
</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Home</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>180</y>
        <width>401</width>
        <height>241</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>icons/joystic.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>440</y>
        <width>491</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 20pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>A simple Python project based on PyQt5</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>440</y>
        <width>491</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 20pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>and TKinter with SQLlite3 database.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>260</y>
        <width>341</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 72pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>mini-PS Project</string>
      </property>
     </widget>
     <widget class="QPushButton" name="doc_btn">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>500</y>
        <width>271</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	color: white;
font: 87 12pt &quot;Arial Black&quot;;
	background-color: rgb(35, 134, 54);
	border-radius:7px;
	border:1px solid #18ab29;
}

QPushButton:hover {
	background:linear-gradient(to bottom, #5cbf2a 5%, #44c767 100%);
	background-color:#18ab29;
}
QPushButton:active {
	position:relative;
	top:2px;
}</string>
      </property>
      <property name="text">
       <string>Project Documentation</string>
      </property>
     </widget>
     <widget class="QPushButton" name="feedback_btn">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>500</y>
        <width>271</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	color: white;
font: 87 12pt &quot;Arial Black&quot;;
	background-color: rgb(35, 134, 54);
	border-radius:7px;
	border:1px solid #18ab29;
}

QPushButton:hover {
	background:linear-gradient(to bottom, #5cbf2a 5%, #44c767 100%);
	background-color:#18ab29;
}
QPushButton:active {
	position:relative;
	top:2px;
}</string>
      </property>
      <property name="text">
       <string>Send Feedback</string>
      </property>
     </widget>
     <widget class="QPushButton" name="github_btn">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>500</y>
        <width>271</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	color: white;
font: 87 12pt &quot;Arial Black&quot;;
	background-color: rgb(35, 134, 54);
	border-radius:7px;
	border:1px solid #18ab29;
}

QPushButton:hover {
	background:linear-gradient(to bottom, #5cbf2a 5%, #44c767 100%);
	background-color:#18ab29;
}
QPushButton:active {
	position:relative;
	top:2px;
}</string>
      </property>
      <property name="text">
       <string>Visit Project Repository</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>AI</string>
     </attribute>
     <widget class="QLabel" name="label_121">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>460</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 12pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>and shoot them using the space button.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_119">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>420</y>
        <width>411</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 12pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>You can control your spaceship with the left and right arrows</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_82">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>521</width>
        <height>571</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>icons/ai.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="play_ai">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>590</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	color: white;
font: 87 15pt &quot;Arial Black&quot;;
	background-color: rgb(35, 134, 54);
	border-radius:7px;
	border:1px solid #18ab29;
}

QPushButton:hover {
	background:linear-gradient(to bottom, #5cbf2a 5%, #44c767 100%);
	background-color:#18ab29;
}
QPushButton:active {
	position:relative;
	top:2px;
}</string>
      </property>
      <property name="text">
       <string>Play</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_118">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>270</y>
        <width>341</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
color: rgb(170, 255, 255);
font: 75 25pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>[~] Alien Invasion Game</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_117">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>340</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 15pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>How to play?</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_120">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>380</y>
        <width>471</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 12pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>As a guardian of the galaxy, you have to  destroy all the spaceships.</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ai_sittings">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>650</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	color: white;
font: 87 15pt &quot;Arial Black&quot;;
	background-color: rgb(35, 134, 54);
	border-radius:7px;
	border:1px solid #18ab29;
}

QPushButton:hover {
	background:linear-gradient(to bottom, #5cbf2a 5%, #44c767 100%);
	background-color:#18ab29;
}
QPushButton:active {
	position:relative;
	top:2px;
}</string>
      </property>
      <property name="text">
       <string>Sittings</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>FB</string>
     </attribute>
     <widget class="QLabel" name="label_86">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>380</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 12pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>This game is based on quick reaction,</string>
      </property>
     </widget>
     <widget class="QPushButton" name="play_fb">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>590</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	color: white;
font: 87 15pt &quot;Arial Black&quot;;
	background-color: rgb(35, 134, 54);
	border-radius:7px;
	border:1px solid #18ab29;
}

QPushButton:hover {
	background:linear-gradient(to bottom, #5cbf2a 5%, #44c767 100%);
	background-color:#18ab29;
}
QPushButton:active {
	position:relative;
	top:2px;
}</string>
      </property>
      <property name="text">
       <string>Play</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_50">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>521</width>
        <height>571</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>icons/bird.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_88">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>420</y>
        <width>471</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 12pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>you have to pass the bird through these pipes using the space button.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_89">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>340</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 15pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>How to play?</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_90">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>270</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
color: rgb(170, 255, 255);
font: 75 25pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>[~] Flappy Bird Game</string>
      </property>
     </widget>
     <widget class="QPushButton" name="fb_sittings">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>650</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	color: white;
font: 87 15pt &quot;Arial Black&quot;;
	background-color: rgb(35, 134, 54);
	border-radius:7px;
	border:1px solid #18ab29;
}

QPushButton:hover {
	background:linear-gradient(to bottom, #5cbf2a 5%, #44c767 100%);
	background-color:#18ab29;
}
QPushButton:active {
	position:relative;
	top:2px;
}</string>
      </property>
      <property name="text">
       <string>Sittings</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>SN</string>
     </attribute>
     <widget class="QLabel" name="label_51">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>270</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
color: rgb(170, 255, 255);
font: 75 25pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>[~] Snake Game</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_52">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>420</y>
        <width>501</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 12pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string> to move the snake to the green target to increase the length of the snake.</string>
      </property>
     </widget>
     <widget class="QPushButton" name="play_sn">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>590</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	color: white;
font: 87 15pt &quot;Arial Black&quot;;
	background-color: rgb(35, 134, 54);
	border-radius:7px;
	border:1px solid #18ab29;
}

QPushButton:hover {
	background:linear-gradient(to bottom, #5cbf2a 5%, #44c767 100%);
	background-color:#18ab29;
}
QPushButton:active {
	position:relative;
	top:2px;
}</string>
      </property>
      <property name="text">
       <string>Play</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_53">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>340</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 15pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>How to play?</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_37">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>521</width>
        <height>571</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>icons/snake_img.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_54">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>380</y>
        <width>391</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 12pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>This game is played by using the four directional buttons,</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sn_sittings">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>650</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	color: white;
font: 87 15pt &quot;Arial Black&quot;;
	background-color: rgb(35, 134, 54);
	border-radius:7px;
	border:1px solid #18ab29;
}

QPushButton:hover {
	background:linear-gradient(to bottom, #5cbf2a 5%, #44c767 100%);
	background-color:#18ab29;
}
QPushButton:active {
	position:relative;
	top:2px;
}</string>
      </property>
      <property name="text">
       <string>Sittings</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>XO</string>
     </attribute>
     <widget class="QLabel" name="label_39">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>380</y>
        <width>361</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 12pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>it's easy game only you need to use your mouse, and</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_40">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>270</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
color: rgb(170, 255, 255);
font: 75 25pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>[~] Tic Tac Toe Game</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_42">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>420</y>
        <width>391</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 12pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>Click in any posation to draw your symbol X or O. enjoy :)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_43">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>521</width>
        <height>571</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>icons/XO_img.PNG</pixmap>
      </property>
     </widget>
     <widget class="QPushButton" name="play_xo">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>590</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	color: white;
font: 87 15pt &quot;Arial Black&quot;;
	background-color: rgb(35, 134, 54);
	border-radius:7px;
	border:1px solid #18ab29;
}

QPushButton:hover {
	background:linear-gradient(to bottom, #5cbf2a 5%, #44c767 100%);
	background-color:#18ab29;
}
QPushButton:active {
	position:relative;
	top:2px;
}</string>
      </property>
      <property name="text">
       <string>Play</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_44">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>340</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 15pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>How to play?</string>
      </property>
     </widget>
     <widget class="QPushButton" name="xo_sittings">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>650</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	color: white;
font: 87 15pt &quot;Arial Black&quot;;
	background-color: rgb(35, 134, 54);
	border-radius:7px;
	border:1px solid #18ab29;
}

QPushButton:hover {
	background:linear-gradient(to bottom, #5cbf2a 5%, #44c767 100%);
	background-color:#18ab29;
}
QPushButton:active {
	position:relative;
	top:2px;
}</string>
      </property>
      <property name="text">
       <string>Sittings</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="home_btn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>201</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/home.png</normaloff>icons/home.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="ai_btn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>201</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/alien.png</normaloff>icons/alien.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="fb_btn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>260</y>
      <width>201</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/bird.png</normaloff>icons/bird.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="sn_btn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>390</y>
      <width>201</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/snake.png</normaloff>icons/snake.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="xo_btn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>520</y>
      <width>201</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/xo.png</normaloff>icons/xo.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="exit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>650</y>
      <width>201</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/exit.png</normaloff>icons/exit.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1363</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="main.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>exit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>666</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>-16</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
